home-data-migration/src/main/java/LiquibaseUpdate.java

```diff
@@ -0,0 +1,66 @@
+import liquibase.Contexts;
+import liquibase.Liquibase;
+import liquibase.database.Database;
+import liquibase.database.DatabaseFactory;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.LiquibaseException;
+import liquibase.resource.FileSystemResourceAccessor;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class LiquibaseUpdate {
+//    name of references in migration.properties
+    private static final String dbhost = "url";
+    private static final String pathname = "changelog.path";
+    private static final String changelog = "changelog.name";
+
+    public void runLiquibase(String user, String password) {
```

Comment:
With a high probability URL of db will vary on the environments. Please use the default if not present approach and give a possibility to specify it.